weatherreport.cpp

```diff
@@ -24,15 +24,15 @@ class SensorStub : public IWeatherSensor {
     }
 
     int Precipitation() const override {
-        return 70;
```

Comment:
This will change the weather sensor, so it's not possible to test in two weathers.
Let this class accept the values in the constructor, so that we can make multiple weather scenarios without having to change the class or make new classes.